### ISSUE TITLE
Frame managers should use Windows not FrameContainers

### ISSUE THREAD

**Issue by greboid** [DMDirc/Plugins#46]
At present some of the Frame Managers (such as the Window menu and the Treeview) tie their internal representations to FrameContainers, and then have to convert back and forth to Windows/TextFrames which is pointless and makes the code hard to follow. They should instead store references to WIndows and use those directly.

**Comment by csmith** [comment]
Obsolete, I think.
